<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Explorer</class>
 <widget class="QMainWindow" name="Explorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Explorer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QListView" name="Win_One"/>
        </item>
        <item>
         <widget class="QListView" name="Win_Two"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="B_Exit">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>445</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
